<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QWidget" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录界面</string>
  </property>
  <widget class="QCheckBox" name="remember_checkbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住账号和密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>301</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="login_button">
      <property name="styleSheet">
       <string notr="true">background-color: #82b74b; color: white; border: none; padding: 8px 15px;</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="exit_button">
      <property name="styleSheet">
       <string notr="true">background-color: #d54d4d; color: white; border: none; padding: 8px 15px;</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="hbox1_2">
      <item>
       <widget class="QLabel" name="server_label">
        <property name="text">
         <string>域  名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="server_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hbox2_2">
      <item>
       <widget class="QLabel" name="db_label">
        <property name="text">
         <string>数据库：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="db_edit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hbox1">
      <item>
       <widget class="QLabel" name="username_label">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hbox2">
      <item>
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>密  码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_edit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_edit</tabstop>
  <tabstop>password_edit</tabstop>
  <tabstop>remember_checkbox</tabstop>
  <tabstop>login_button</tabstop>
  <tabstop>exit_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>login_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>loginsignal1()</signal>
  <slot>login()</slot>
 </slots>
</ui>
